BSP/Inc/BSP_GPIO.h

```diff
@@ -55,7 +56,7 @@ uint8_t BSP_GPIO_Read_Pin(port_t port, uint16_t pinmask);
  * @param   state ON or OFF
```

Comment:
param is now a true or false value so the documentation can reflect that

Reply:
Nathaniel says you're wrong (and not to change anything in BSP)